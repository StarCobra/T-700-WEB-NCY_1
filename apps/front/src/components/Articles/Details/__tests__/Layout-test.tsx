import React from "react";
import { act, create } from "react-test-renderer";
import Component from "../Layout";

describe("Components > Articles > Details > Layout", () => {
  const tests = [
    {
      description: "generic",
      props: {},
    },
  ];
  tests.forEach((test) => {
    it(`should render the component ${test.description}`, async () => {
      let root;
      await act(async () => {
        root = create(<Component {...test.props} />);
      });
      expect(root).toMatchSnapshot();
    });
  });
});
