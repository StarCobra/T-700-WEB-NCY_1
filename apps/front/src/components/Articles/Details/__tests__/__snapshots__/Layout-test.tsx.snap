// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Articles > Details > Layout should render the component generic 1`] = `
<div
  className="fullArticleDisplay MuiBox-root css-0"
>
  <div
    className="articleHeader MuiBox-root css-0"
  >
    <div
      className="articleThumbnail MuiBox-root css-0"
    >
      <img
        alt=""
        src="default"
      />
    </div>
    <div
      className="articleTitleContainer MuiBox-root css-0"
    >
      <div
        className="title MuiBox-root css-0"
      >
        <h3>
          Bitcoin Collapse
        </h3>
      </div>
      <div
        className="dateAndSource MuiBox-root css-0"
      >
        <h4>
          From ChatGPT the 2023-11-13
        </h4>
      </div>
    </div>
  </div>
  <div
    className="fullArticleContent MuiBox-root css-0"
  >
    <div
      className="contentContainer MuiBox-root css-0"
    >
      <p>
        LOREM IPSUM
      </p>
    </div>
    <div
      className="contentContainer MuiBox-root css-0"
    >
      <h3>
        Regulatory Challenges:
      </h3>
      <p>
        LOREM IPSUM
      </p>
    </div>
    <div
      className="contentContainer MuiBox-root css-0"
    >
      <h3>
        Environmental Concerns:
      </h3>
      <p>
        LOREM IPSUM
      </p>
    </div>
    <div
      className="contentContainer MuiBox-root css-0"
    >
      <h3>
        Technological Challenges:
      </h3>
      <p>
        LOREM IPSUM
      </p>
    </div>
    <div
      className="contentContainer MuiBox-root css-0"
    >
      <h3>
        Market Volatility:
      </h3>
      <p>
        LOREM IPSUM
      </p>
    </div>
    <div
      className="contentContainer MuiBox-root css-0"
    >
      <h3>
        Conclusion:
      </h3>
      <p>
        LOREM IPSUM
      </p>
    </div>
  </div>
</div>
`;
